ros2_batch_job/__main__.py

```diff
@@ -273,7 +283,56 @@ def get_args(sysargv=None):
     return args
 
 
-def process_coverage(args, job):
+def include_python_coverage_in_report(args, package=None):
+    # no arguments means report all packages in buildspace
+    root_directory_to_report = args.buildspace
+    if package:
+        root_directory_to_report = os.path.join(root_directory_to_report, package)
+
+    coverage_xml_files = glob.glob(
+        os.path.join(root_directory_to_report, '**', 'coverage.xml'),
+        recursive=True)
+    for coverage_xml_path in coverage_xml_files:
+        # python coverage detected: move the coverage.xml file to buildspace to be reported
```

Comment:
Nitpick: while the comment says "move" the code actually copies.

Reply:
ec4acd89415d644be33c34457bf09ea1f019d754